openedx/core/djangoapps/content_libraries/tests/test_content_libraries.py

```diff
@@ -974,6 +976,52 @@ def test_library_block_delete_event(self):
             event_receiver.call_args.kwargs
         )
 
+    def test_library_paste_clipboard(self):
+        """
+        Check the a new block is created in the library after pasting from clipboard.
+        The content of the new block should match the content of the block in the clipboard.
+        """
+        # Importing here since this was failing when tests ran in the LMS
```

Comment:
This whole class is annotated with `@skip_unless_cms` so these particular tests shouldn't run in the LMS. I guess what happened was that the file was still _imported_ when the LMS test suite ran, and that's what caused this error?

Reply:
> the file was still imported when the LMS test suite ran, and that's what caused this error?

Yes that is indeed what is happening, the comment should probably be made more clear.